sources/dogtag/Cargo.toml

```
@@ -0,0 +1,30 @@
+[package]
+name = "dogtag"
+version = "0.1.0"
+authors = ["Jarrett Tierney <jmt@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+exclude = ["README.md"]
+
+[[bin]]
+name = "20-imds"
+path = "bin/imds.rs"
+
+[[bin]]
+name = "10-reverse-dns"
+path = "bin/reverse.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+argh = "0.1"
+dns-lookup = "2"
+imdsclient = { version = "0.1", path = "../imdsclient" }
+log = "0.4"
+snafu = "0.8"
+tokio = { version = "1.32", features = ["full"]}
+walkdir = "2.4"
```

```suggestion
tokio = { version = "~1.32", features = ["full"]}  # LTS
walkdir = "2"
```

Also, do you need the full feature set here?

---

No but we use full everywhere else

---

That's not the case for crates under `/sources`.